README.md

```
@@ -1,381 +1,156 @@
 ```
 
-手動同期では、PRの取得、コメント・差分の収集、要約・分析が実行されます。
-
-### バックアップ
+#### 別ペインでテスト実行
 
-```bash
-make backup
-# または
-curl -X POST http://localhost:8081/api/v1/backup
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)